dev/release/post-14-homebrew.sh

```diff
@@ -40,10 +40,12 @@ fi
 echo "Updating working copy"
 git fetch --all --prune --tags --force -j$(nproc)
 
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SOURCE_DIR}/git-vars.sh"
 branch=apache-arrow-${version}
 echo "Creating branch: ${branch}"
 git branch -D ${branch} || :
-git checkout -b ${branch} origin/master
+git checkout -b ${branch} origin/${DEFAULT_BRANCH}
```

Comment:
We should not change this because the repository is a fork of Homebrew/homebrew-core not apache/arrow.